src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSortVariant.jsx

```diff
@@ -80,19 +77,40 @@ export default function ProviderSortVariant({
   const hasUserAddress = address && !!Object.keys(address).length;
   const requestLocationStatusFailed =
     requestLocationStatus === FETCH_STATUS.failed;
+
+  if (hasUserAddress) {
+    sortOptions.unshift({
+      value: FACILITY_SORT_METHODS.distanceFromResidential,
+      label: 'Your home address',
+    });
+  } else {
+    sortOptions.push({
+      value: FACILITY_SORT_METHODS.distanceFromCurrentLocation,
+      label: 'Your current location',
+    });
+  }
+
   return (
     <div className="vads-u-margin-bottom--3">
       {notLoading && (
-        <p
-          className="vads-u-margin--0"
-          id="provider-list-status"
-          role="status"
-          aria-live="polite"
-          aria-atomic="true"
-        >
-          Displaying 1 to {currentlyShownProvidersList.length} of{' '}
-          {communityCareProviderList.length} providers
-        </p>
+        <>
+          <p
+            className="vads-u-margin--0"
+            id="provider-list-status"
+            role="status"
+            aria-live="polite"
+            aria-atomic="true"
+          >
+            Displaying 1 to {currentlyShownProvidersList.length} of{' '}
+            {communityCareProviderList.length} providers
+          </p>
+          <h3 className="vads-u-font-size--h5">
```

Comment:
My interpretation of the copy doc is that these "on file" headings for the doc, not content that should go in the page, and that we don't display an address. Maybe check with @outerpress to make sure.

Reply:
Done.